mendix/web-widgets
# [WC-1398]: Renaming data-widget scss files to avoid duplication

### Description

Because files previously did not start with and underscore, scss compiler included those in final css bundle. To prevent this and follow file naming convention we renamed files and added underscore to the file names.

### Pull request type



-   [ ] No code changes (changes to documentation, CI, metadata, etc)
-   [ ] Test related change (New E2E test, test automation, etc.)

### What should be covered while testing?


Steps to test:
1. Create a new app from blank template
2. Run the app to get deployment directory
3. Copy the `deployment/web/theme.compiled.css` to `/tmp/old.css`
4. Create a new project from blank template
5. Go to `themesource/datawidgets` and remove all the files.
6. Build the styles from this branch, and copy them inside `themesource/datawidgets` 
7. Run the app.
8. Copy the `deployment/web/theme.compiled.css` to `/tmp/new.css`
9. Compare `old.css` with `new.css`

Expected result:
`new.css` should remove the duplicated css rules.
